bedrock/urls_mozorg_mode.py

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from django.conf import settings
```

Comment:
Since we import settings in `urls.py` should we just keep a single `urls.py` and add whichever URL configs are appropriate for the mode to a base set that's common to both?

Reply:
Yeah that's a good idea - simpler

- [ ] @stevejalim Flip it so we're making a choice within a single urls.py, rather than choosing between two `urls.py`s

Reply:
(Reflecting: adding some conditional additions to a single urls.py is fine for now, deffo, but if we start using bedrock to serve a bunch of different sites, that file might get a bit hefty. Not an issue right now, though)

Reply:
I was thinking about this over the weekend, and I think it's better if, rather than have a few scatterde if/else blocks that check SITE_MODE, it's cleaner to just have one in `settings/__init__.py` so it's easier to see what's being turned on/off or changed for each mode. See what y'all think when I mark this for re-review shortly.